addon/services/breadcrumbs.js

```diff
@@ -15,6 +15,40 @@ export default Service.extend({
   init () {
     this._super(...arguments)
 
-    this.instances = A([])
+    this.containers = []
+  },
+
+  /**
+   * Methods
+   */
+
+  registerContainer (containerData) {
+    assert(
+      'An `element` is required to register a breadcrumbs container',
+      containerData.element
+    )
+    assert(
+      'This container was already registered before',
+      !this.isContainerRegistered(containerData)
+    )
+
+    this.set('containers', [...this.containers, containerData])
+  },
+
+  unregisterContainer ({ element }) {
+    this.set(
+      'containers',
+      this.containers.filter(registeredContainer => {
+        return element === registeredContainer.element
```

Comment:
`!==` no?

Reply:
heh, that's what happens when you don't write tests :P